modules/datastore/src/Controller/QueryController.php

```diff
@@ -49,7 +58,8 @@ public function __construct(Service $datastoreService, RequestStack $requestStac
   public static function create(ContainerInterface $container) {
     $datastoreService = $container->get('dkan.datastore.service');
     $requestStack = $container->get('request_stack');
-    return new QueryController($datastoreService, $requestStack);
+    $datasetInfo = $container->get('dkan.common.dataset_info');
```

Comment:
I think this is not the ideal service to use here, but I know it's the easiest one. Just noting that we want to consolidate DatasetInfo with the metastore classes.